code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -1077,6 +1076,10 @@
 		target.deploy_init(src)
 		mind.transfer_to(target)
 	diag_hud_set_deployed()
+	//bubber edit begin
+	var/mob/living/silicon/ai/AI = src
+	AI.in_shell = TRUE
```

Comment:
```suggestion
	in_shell = TRUE
```
Should work(tm)

Reply:
But honestly you can just check if deployed_shell is null or not, that'll take care of this

Reply:
done